epersonregistrations.md

```diff
@@ -44,13 +44,16 @@ To create a new EPerson registration, perform a post with the JSON below to the
 }
 ```
 
+Requires query parameter 'type' set with either 'register' or 'forgot' value, depending on the action requested.
```

Comment:
this was confusing for me at first read as there is an overlapping between the type in query string and the type in the json payload. We could try to highlight this difference in the text or we could use a different query param name or we could offer as option "new" or "forgot" instead than "register" or "forgot" so that the json `type:'register'` is not confused with that

Reply:
Changed to `accountRequestType` to avoid confusion